app/src/main/java/com/puzzlebench/clean_marvel_kotlin/data/service/CharacterServicesImpl.kt

```diff
@@ -13,9 +13,21 @@ class CharacterServicesImpl(private val api: MarvelResquestGenerator = MarvelRes
         return Observable.create { subscriber ->
```

Comment:
Please don't use create methods. Use fromCallable. Also, you can use Single instead Observable.